package.json

```diff
@@ -2,13 +2,14 @@
   "name": "msteams-react-base-component",
   "version": "2.1.0",
   "description": "React helper components for Microsoft Teams apps",
-  "main": "lib/index.js",
-  "types": "lib/index.d.ts",
+  "main": "lib/cjs/index.js",
+  "module": "lib/esm/index.js",
+  "types": "lib/cjs/index.d.ts",
   "scripts": {
     "prepublishOnly ": "npm run clean && npm run build",
-    "build": "node_modules/.bin/tsc -p .",
-    "clean": "rm -rf lib",
-    "lint": "./node_modules/.bin/eslint ./src/**/*.tsx"
+    "build": "tsc -p tsconfig.json && tsc -p tsconfig.es.json",
+    "clean": "rimraf lib",
```

Comment:
![image](https://user-images.githubusercontent.com/14183168/97278608-46354c80-183a-11eb-8cfc-b3cfd0aa3931.png)

There is no `rm` on Windows 😸

Reply:
there is no rimraf on windows as well )
`rmdir /q/s lib`